stable/mattermost-team-edition/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 2.2.1
+version: 2.3.0
```

Comment:
since is just a small fix, just the minor
```suggestion
version: 2.2.2
```

Reply:
it's small fix but changes to user experience (push wasn't working because it was trying to use enterprise push server)

Reply:
yep, but sounds more a bug fix

Reply:
@ahmadalli thanks. I will fix the DCO